src/Lib.hs

```diff
@@ -335,21 +335,30 @@ inspectAttachments ticketInfo attachments = runMaybeT $ do
     lastAttachment  <- MaybeT . pure $ lastAttach
     att             <- MaybeT $ getAttachment lastAttachment
 
-    pure $ inspectAttachment config ticketInfo att
+    catchAny (inspectAttachment config ticketInfo att)
```

Comment:
We are not looking for `Any` exception, we need specific exceptions we can throw and catch. Like here - https://github.com/input-output-hk/cardano-sl/blob/feature/CBR-163/wallet-new/src/Cardano/Wallet/WalletLayer/Error.hs

Reply:
Thanks for the advice! Will look into it!